test-projects/access-control/cypress/integration/field/admin-ui.js

```diff
@@ -152,42 +203,55 @@ describe('Access Control Fields > Admin UI', () => {
           });
       });
 
-      // Skipped until Admin UI hides these fields
-      it.skip('does not show non-updatable inputs', () => {
+      it('does not show non-updatable inputs', () => {
         fieldAccessVariations
           .filter(({ update, read }) => !update && read)
           .forEach(access => {
             const field = getFieldName(access);
             cy.get(`label[for="ks-input-${field}"]`)
-              .should('exist')
+              .should('not.exist')
               .then(() => cy.get(`#ks-input-${field}`).should('not.exist'));
           });
       });
-    }
-
-    function updateFromListViewTests() {
-      describe('list view multi-update', () => {
-        it.skip('shows only the fields commonly updatable across all selected items', () => {
-          // TODO: Setup access control so that it's something like this:
-          //
-          // | Is Updatable | foo | bar | zip |
-          // |--------------|-----|-----|-----|
-          // | Item 1       |  ✓  |     |  ✓  |
-          // | Item 2       |     |  ✓  |  ✓  |
-          //
-          // And check that the updatable fields are the intersection (ie; only
-          // 'zip' in this case)
-        });
-      });
-    }
 
-    describe(`static on ${staticList}`, () => {
-      updateTests(staticList);
       updateFromListViewTests(staticList);
     });
 
     describe(`imperative on ${imperativeList}`, () => {
-      updateTests(imperativeList);
+      const slug = listSlug(imperativeList);
+      const queryName = `all${imperativeList}s`;
+
+      before(() => {
+        cy.graphql_query('/admin/api', `query { ${queryName}(first: 1) { id } }`).then(
+          ({ data }) => {
+            cy.visit(`/admin/${slug}/${data[queryName][0].id}`);
+            cy.get('form').should('exist');
+          }
+        );
+      });
+
+      it('shows updatable inputs', () => {
+        fieldAccessVariations
+          .filter(({ update, read }) => update && read)
+          .forEach(access => {
+            const field = getFieldName(access);
+            cy.get(`label[for="ks-input-${field}"]`)
+              .should('exist')
+              .then(() => cy.get(`#ks-input-${field}`).should('exist'));
+          });
+      });
+
+      it.skip('does not show non-updatable inputs', () => {
+        fieldAccessVariations
+          .filter(({ update, read }) => !update && read)
+          .forEach(access => {
+            const field = getFieldName(access);
+            cy.get(`label[for="ks-input-${field}"]`)
+              .should('not.exist')
+              .then(() => cy.get(`#ks-input-${field}`).should('not.exist'));
+          });
+      });
+
```

Comment:
I unrolled these tests as I needed to `.skip` specific ones instead of whole categories of tests.